src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +26,14 @@
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
 
+    void fill(List<Object> machines, Machine value);
```

Comment:
The `fill` method should have a bounded wildcard for the `machines` list to ensure type safety. Use `List<? super T>` to allow lists of any supertype of `T`.